packages/block-library/src/accordions/edit.js

```diff
@@ -35,49 +42,90 @@ export default function Edit( {
-					<ToggleControl
-						isBlock
-						__nextHasNoMarginBottom
+				<ToolsPanel
+					label={ __( 'Settings' ) }
+					resetAll={ () => {
+						setAttributes( {
+							autoclose: false,
+							showIcon: true,
+							iconPosition: 'right',
+						} );
+					} }
+					dropdownMenuProps={ dropdownMenuProps }
+				>
+					<ToolsPanelItem
 						label={ __( 'Auto-close' ) }
-						onChange={ ( value ) => {
-							setAttributes( {
-								autoclose: value,
-							} );
-						} }
-						checked={ autoclose }
-						help={ __(
-							'Automatically close accordions when a new one is opened.'
-						) }
-					/>
-					<ToggleControl
-						isBlock
-						__nextHasNoMarginBottom
+						isShownByDefault
+						hasValue={ () => !! autoclose }
+						onDeselect={ () =>
+							setAttributes( { autoclose: false } )
+						}
+					>
+						<ToggleControl
+							isBlock
+							__nextHasNoMarginBottom
+							label={ __( 'Auto-close' ) }
+							onChange={ ( value ) => {
+								setAttributes( {
+									autoclose: value,
+								} );
+							} }
+							checked={ autoclose }
+							help={ __(
+								'Automatically close accordions when a new one is opened.'
+							) }
+						/>
+					</ToolsPanelItem>
+					<ToolsPanelItem
 						label={ __( 'Show icon' ) }
-						onChange={ ( value ) => {
-							setAttributes( {
-								showIcon: value,
-							} );
-						} }
-						checked={ showIcon }
-						help={ __(
-							'Display a plus icon next to the accordion header.'
-						) }
-					/>
-					<ToggleGroupControl
-						__nextHasNoMarginBottom
-						__next40pxDefaultSize
-						isBlock
+						isShownByDefault
+						hasValue={ () => ! showIcon }
+						onDeselect={ () => setAttributes( { showIcon: true } ) }
+					>
+						<ToggleControl
+							isBlock
+							__nextHasNoMarginBottom
+							label={ __( 'Show icon' ) }
+							onChange={ ( value ) => {
+								setAttributes( {
+									showIcon: value,
+								} );
+							} }
+							checked={ showIcon }
+							help={ __(
+								'Display a plus icon next to the accordion header.'
+							) }
+						/>
+					</ToolsPanelItem>
+					<ToolsPanelItem
 						label={ __( 'Icon Position' ) }
-						value={ iconPosition }
-						onChange={ ( value ) => {
-							setAttributes( { iconPosition: value } );
-						} }
+						isShownByDefault
+						hasValue={ () => iconPosition !== 'right' }
+						onDeselect={ () =>
+							setAttributes( { iconPosition: 'right' } )
+						}
 					>
-						<ToggleGroupControlOption label="Left" value="left" />
-						<ToggleGroupControlOption label="Right" value="right" />
-					</ToggleGroupControl>
-				</PanelBody>
+						<ToggleGroupControl
+							__nextHasNoMarginBottom
+							__next40pxDefaultSize
+							isBlock
+							label={ __( 'Icon Position' ) }
+							value={ iconPosition }
+							onChange={ ( value ) => {
+								setAttributes( { iconPosition: value } );
+							} }
+						>
+							<ToggleGroupControlOption
+								label="Left"
```

Comment:
```suggestion
								label={ __( 'Left' ) }
```

Although it's not relevant to the purpose of this PR, let's localize the labels.